assets/section-signature-bracelet.css

```diff
@@ -0,0 +1,89 @@
+.signature-bracelet__pre-heading {
+  font-size: 1.4rem;
+  font-weight: 700;
+  text-transform: uppercase;
+  color: #2E2F38;
+  letter-spacing: 0.2em;
+  line-height: 1.7;
+}
+.signature-bracelet__desc {
+  font-size: 1.6rem;
+  font-weight: 500;
+  line-height: 1.5;
+  margin-bottom: 3.8rem;
+}
+.signature-bracelet__desc .rte {
+  display: inline-block;
+  max-width: 660px;
+}
+.signature-bracelet__desc strong {
+  color: #0565CE;
+}
+.signature-bracelet__grid-main-image {
+  margin-bottom: 3rem;
+}
+.signature-bracelet__grid-main-image img {
+  width: 100%;
+  height: auto;
+}
+.signature-bracelet__gird-icon-text {
+  font-size: 1.4rem;
+}
+.signature-bracelet__product {
+  max-width: 370px;
+  margin: 4rem auto 0;
+}
+.signature-bracelet__product .product-form__submit {
+  font-size: 1.6rem;
+  font-weight: 700;
+  min-height: 60px;
+}
+
+@media screen and (max-width: 749px) {
+  .signature-bracelet__gird-icon {
+    max-width: 280px;
+    margin: 0 auto 1.8rem;
+    display: flex;
+    align-items: center;
+  }
+  .signature-bracelet__gird-icon.hidden-mobile {
+    display: none;
+  }
+  .signature-bracelet__gird-icon-image {
+    flex: 0 0 4.3rem;
+    margin-right: 1.7rem;
+  }
+}
+
+@media screen and (min-width: 750px) {
+  .signature-bracelet__pre-heading {
+    font-size: 1.8rem;
+  }
+  .signature-bracelet__desc {
+    font-size: 1.8rem;
+    line-height: 1.7;
+    margin-bottom: 6.2rem;
+  }
+  .signature-bracelet__heading {
+    margin-top: 0;
+    margin-bottom: 2.3rem;
+  }
+  .signature-bracelet__gird-icon {
+    text-align: center;
+  }
+  .signature-bracelet__gird-icon-image {
```

Comment:
In mock it is centered on mobile. Add justify-self: center.

Reply:
Not sure what you mean, and why add this. Is it not centered for you on mobile?

Reply:
There are style for mobile only added for this element